lib/task/task.js

```diff
@@ -45,23 +46,27 @@ const s3 = require('../external/s3').init(config.get('default.external.s3blobSto
 
 const task = {
   // not thread-safe! but we don't have threads..
-  withContainer: (taskdef) => (...args) => {
+  withContainer: (name, taskdef) => (...args) => {
     const needsContainer = (task._container == null);
     if (needsContainer) task._container = container.withDefaults({ db: slonikPool(config.get('default.database')), env, mail, task: true, odkAnalytics, s3 });
 
-    const result = taskdef(task._container)(...args);
-
-    // early return with a modified result chain if cleanup is needed:
-    if (needsContainer) {
-      return result.finally(() => {
-        const { db } = task._container;
-        delete task._container;
-        return db.end();
-      });
-    }
-
-    // otherwise just return.
-    return result;
+    return Sentry.startSpan(
+      { op: `task.${name}`, name: JSON.stringify(process.argv) },
```

Comment:
Is there a chance that `process.argv` will include sensitive information that shouldn't end up on Sentry? For example, I think it could contain an email address? We generally don't share request bodies with Sentry. Not sure if that's a fair comparison, but I'm curious about what could go into `process.argv`.

Reply:
Yes, it's possible.

I'll remove `process.argv`.